docs: update getting started app command guides (#202)
sapphiredev/website: docs/Guide/application-commands/what-are-application-commands.mdx

---
sidebar_position: 0
title: What are application commands?
---

If you've been around for a while, you know that the conventional way of interacting with Discord bots is through
messaging, either by pinging one or starting a message with a prefix like `$` or `!`.

> These days, setting up [Application Commands][discord-application-command-docs] are now recommended for Discord bots.

Reviewer: ```suggestion
These days, setting up [Application Commands][discord-application-command-docs] is recommended for Discord bots.
```

> Instead of worrying about issues like prefix conflicts or argument parsing, Discord now provides native interfaces to

Reviewer: ```suggestion
Instead of worrying about issues like prefix conflicts or argument parsing, Discord now provides a native interfaces to
```

interact with your bot in the form of slash commands, message context commands, and user context commands.

:::info

If your bot isn't verified (a requirement to be in 100+ servers), then you aren't required to set up application

> commands and can still use classical message commands. This has been a long change in the making that Discord discusses

Reviewer: ```suggestion
commands and can still use legacy message commands. This has been a long change in the making that Discord addresses
```
